modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -1093,7 +1093,16 @@ void GDScriptAnalyzer::resolve_class_member(GDScriptParser::ClassNode *p_class,
 							}
 						}
 						if (is_get_node) {
-							parser->push_warning(member.variable, GDScriptWarning::GET_NODE_DEFAULT_WITHOUT_ONREADY, is_using_shorthand ? "$" : "get_node()");
+							GDScriptParser::GetNodeNode *get_node_node = static_cast<GDScriptParser::GetNodeNode *>(expr);
```

Comment:
I'd move this into the `if (is_using_shorthand)` condition. Otherwise it's an incorrect cast if the type is not `GET_NODE`.

Reply:
Done, thanks!